google/resource_compute_instance_test.go

```diff
@@ -2902,10 +2906,17 @@ resource "google_compute_instance" "foobar" {
   network_interface {
     subnetwork = "${google_compute_subnetwork.inst-test-subnetwork.self_link}"
 
-    alias_ip_range {
-      subnetwork_range_name = "${google_compute_subnetwork.inst-test-subnetwork.secondary_ip_range.0.range_name}"
-      ip_cidr_range         = "172.16.0.0/24"
-    }
+    alias_ip_range = [
+			{
```

Comment:
some weird indentation here but feel free to ignore unless it really bothers you

Reply:
Fixed throughout.  Vim magic:
`:%s/^\(^I\)*  /\1^I/gc`